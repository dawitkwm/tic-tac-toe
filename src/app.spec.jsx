import React from "react";
import { shallow } from "enzyme";

import App from "./app";

describe("App component", () => {
  it("should render game App component", () => {
    const appWrapper = shallow(<App />);
    expect(appWrapper).toMatchSnapshot();
  });

  it("player x should start the game", () => {
    const appWrapper = shallow(<App />);
    expect(appWrapper.state("xTurn")).toEqual(true);
  });
});
