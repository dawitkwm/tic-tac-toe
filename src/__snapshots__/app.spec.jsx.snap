// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render game App component 1`] = `""`;
